Zend/zend_exceptions.h

```diff
@@ -75,8 +75,8 @@ ZEND_API ZEND_COLD zend_object *zend_create_unwind_exit(void);
 ZEND_API ZEND_COLD zend_object *zend_create_graceful_exit(void);
 ZEND_API ZEND_COLD void zend_throw_unwind_exit(void);
 ZEND_API ZEND_COLD void zend_throw_graceful_exit(void);
-ZEND_API bool zend_is_unwind_exit(const zend_object *ex);
-ZEND_API bool zend_is_graceful_exit(const zend_object *ex);
+ZEND_API ZEND_ATTRIBUTE_PURE bool zend_is_unwind_exit(const zend_object *ex);
```

Comment:
could you elaborate the benefits for those two for example ? while I see clearly for `i_get_exception_base` I m less sure here.